assessment/libs/simplex.php

```diff
@@ -291,15 +291,18 @@ function simplexchecksolution($type,$HasObjective,$solutionlist,$stuanswer) {
                         $dec1 = fractiontodecimal($solutionlist[$r][$c]);
                     }
 
-                    if(is_array($stuanswer[$LastStuColumn])) {
-                        $dec2 = $stuanswer[$c][0]/$stuanswer[$c][1];
-                    } else {
-                        $dec2 = fractiontodecimal($stuanswer[$c]);
-                    }
+					if(array_key_exists($LastStuColumn, $stuanswer)) {
```

Comment:
If the key doesn't exist, should $match be set =0 to indicate there's not a solution?